lib/provisioning/ProvisionDispatcher.js

```diff
@@ -28,32 +28,39 @@ class ProvisionDispatcher {
      * @constructor
      * @param {object} zkConfig - zookeeper config object
      * @param {string} zkConfig.connectionString - zookeeper connection string
+     * @param {string} _zookeeper - zookeeper module
```

Comment:
minor: you can specify those two params are optional: `[_zookeeper]`, `[_runningCI]`